### PR TITLE
Add homepage

### PR DESCRIPTION
**Pull request by iamsjunaid**
### In this milestone I have build the following

- _**International Conference Homepage**_

using the following technologies

1. _HTML_

2. _CSS_

3.  _JavaScript_

4.  _Bootstrap_

### CODE REVIEW

**Review by shella12 — COMMENTED** [review]
Hi @iamsjunaid , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have linters setup properly. Please make sure to setup linters for [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) using the link. Currently you only have linters setup for `JS/css`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by shella12 — COMMENTED** [review]
Hi @iamsjunaid , 

#  Status: Invalidated :red_circle: 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that 

- [ ] doesn't have linters setup properly. Please make sure to setup linters for [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) using the link. Currently you only have linters setup for JS/css. For this project there should at least five linter checks. Currently you are missing `webhint`.  

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by iamsjunaid** [comment]
> Hi @iamsjunaid ,
> 
> # Status: Invalidated 🔴
> While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
> 
> ## Invalid Code Review Request
> You have submitted a project, that
> 
> * [ ]  doesn't have linters setup properly. Please make sure to setup linters for [html-css-js](https://github.com/microverseinc/linters-config/tree/master/html-css-js) using the link. Currently you only have linters setup for JS/css. For this project there should at least five linter checks. Currently you are missing `webhint`.
> 
> Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by iamsjunaid** [comment]
I accidentally closed the pull request.

**Comment by iamsjunaid** [comment]
Hi @shella12 can you please review my pull request again as I am waiting for code review from last two hours?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                    theories, and equations.
+                </p>
+            </div>
+        </div>
+        <a href="#" class="program-details-link m-4">SEE THE WHOLE PROGRAM</a>
+        <button class="join-btn" type="button">Join the Conference 2023</button>
+    </section>
+    <section class="speakers-section">
+        <h2 class="d-flex flex-column justify-content-center align-items-center mb-5">
+            Featured speakers <span class="underline"></span>
```
Comment (Alfredbis29) [inline index.html]: - [ ] As per the project requirements, this section of the project needs to be rendered dynamically using Javascript, which requires storing the details of all the speakers in an array of objects. To achieve this, it is recommended to create a function that iterates through the speakers' list on window load and creates HTML elements with the data from each object. If you need a better understanding of how to achieve this, you can refer to this  [documentation](Best Ways to Create Dynamic HTML Element & CSS in JavaScript)

Furthermore, please ensure that this section of the project is styled in accordance with the design template. It is essential to stick to the design layout, including spacing, margins, colors, and other design elements, to ensure that the section looks professional and visually appealing. Thank you for your attention to these details, which will help to make this project a success.

Your Design :point_down: 
Screenshot from 2023-04-20 09-47-36

Template Design 
 wwgw4g

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+                <i class="fa-brands fa-facebook-f"></i>
+                <i class="fa-brands fa-twitter"></i>
+                <p class="m-0">English</p>
+                <p class="m-0">My Page</p>
+                <p class="m-0">Logout</p>
+            </div>
+        </div>
+        <div class="d-flex justify-content-between align-items-center navbar-style px-4">
+            <a class="logo" href="./index.html">
+                <img src="https://isete.org/images/logo.png" alt="ISETE Logo" />
```
Comment (Alfredbis29) [inline index.html:33]: - [ ] Good job so far I can see that on the right side of the nave we do have the name of the company, kindly complete that part for your project to match the template design :wink: ,  Also, for the bottom navbar, please make sure that the margin on the left-hand side and right-hand side are equal. 
Your design :point_down:  and when clicked to one of your pages others disappearing
Screenshot from 2023-04-20 10-04-35

Template design :point_down: 
dyjxrtj

_note: I can see that you are missing some pages on your nave bar such as Pograms,.._

**Review by Alfredbis29 — CHANGES_REQUESTED** [review]
#  STATUS: REQUIRED CHANGES 
 ## Hi @iamsjunaid ,
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  
 ## Highlights 
 :heavy_check_mark: Linters are passing.
:heavy_check_mark: You have a good code structure.
 
 ## Required Changes 

- [ ] As required for this project You should  create a professional 5-minute video showcasing the project's features, providing a clear and concise description in professional English, demonstrating the project's pages in different screen sizes, and highlighting a piece of code they found interesting or are proud of making. :wink:  KIndly record a video 
- [ ] You are provinding this link as a live demo but when open it is the video record [Video Record](https://www.loom.com/share/f6f7f9969fa344c5ad90b58838e6d1a6) , kindly deploy your project and add the live demo in your readme as well :wink: 

_Check Under the Review
 
 ## Optional suggestions 

- [ ]N/A

 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
 
 Cheers and Happy coding! :clap: :clap: :clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (OyePriscilla) [reply index.html:33]: - [ ] Please kindly add the missing `nav items (Program, Join, Sponsor and News)` according to the given template and as raised earlier. 👍

File: index.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    Researchers will convey critical information, history, background,
+                    theories, and equations.
+                </p>
+            </div>
+        </div>
+        <a href="#" class="program-details-link m-4">SEE THE WHOLE PROGRAM</a>
+        <button class="join-btn" type="button">Join the Conference 2023</button>
+    </section>
+    <section class="speakers-section">
+    </section>
```
Comment (OyePriscilla) [inline index.html:139]: - [ ] Awesome rendering the content of this section dynamically with JS as expected, but please kindly design it according to the given template. 👍 

## Given template
Untitled

## Your Design
Untitled

File: README.md [hunk 4]
```diff
@@ -1,2 +1,201 @@
...
+- **Responsive**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://www.loom.com/share/f6f7f9969fa344c5ad90b58838e6d1a6)
+
```
Comment (OyePriscilla) [inline README.md]: - [ ] Kindly add a live demo as raised earlier, this will make your README more engaging. 👍

**Review by OyePriscilla — CHANGES_REQUESTED** [review]
Hi @iamsjunaid 👏 👏 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

so-close-so-close-seth-meyers

 ## Highlights
Use Flexbox to place elements in the page ✔️
Use JavaScript to manipulate DOM elements ✔️
Use JavaScript events ✔️
Use objects to store and access data ✔️

 ## Required Changes ♻️

- [ ] Awesome recording video, can you please highlight a piece of code you that's interesting to you according to the requirement. Thanks 👍 

Untitled

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @iamsjunaid , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

- [ ] [optional] @iamsjunaid Great work on your project, however, while browsing through your project I noted a bug that appears once in a while, I couldn't identify the source but it would be nice if you tried to fix it 
image



_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**